packit_service/worker/handlers/github_handlers.py

```diff
@@ -211,8 +211,8 @@ def run(self) -> TaskResults:
                 f"| dist-git branch | error |\n"
                 f"| --------------- | ----- |\n"
                 f"{branch_errors}\n\n"
-                "You can re-trigger the update by adding `/packit propose-update`"
-                " to the issue comment.\n"
+                "You can re-trigger the update by adding a comment (`/packit propose-update`)"
```

Comment:
could you please change to `retrigger`? #grammarPolice

Reply:
changed